lib/handlers/quote/quote.ts

```diff
@@ -63,6 +63,7 @@ export class QuoteHandler extends APIGLambdaHandler<
         metric,
       },
     } = params
+    metric.putMetric(`GET_Quote_Requested_ChainId: ${chainId}`, 1, MetricLoggerUnit.Count)
```

Comment:
Should we make these metrics all caps? Not sure if we are following a pattern here.
```suggestion
    metric.putMetric(`GET_QUOTE_REQUESTED_CHAIN_ID: ${chainId}`, 1, MetricLoggerUnit.Count)
```

Reply:
Yeah I can change these

Reply:
I made them all caps